README.md

```diff
@@ -86,15 +86,18 @@ Shipwright supports any tool that can build container images in Kubernetes clust
   ```
 
   ```bash
-  $ kubectl get builds
```

Comment:
Here I think we should leave the `$` sign, it shows it was a copy-and-paste of an command-output.

Reply:
thanks @otaviof for the feedback.

Do you mean leave the `$` sign for the following code block
```
NAME                              SUCCEEDED   REASON      STARTTIME   COMPLETIONTIME
buildpack-nodejs-buildrun-xyzds   True        Succeeded   69s         2s
```

and not `kubectl get builds`? Since the _kubectl_ command is the actual command the user will run, a `$` at the beginning of this command will fail the execution of the command.

I agree on keeping `$` for the command output and will make the change once I get your confirmation.

Reply:
The idea is to show what has happened. In the context, the user has ran `kubectl get builds` and the text document the outcomes the user should observe. In other words:

```text
$ kubectl get builds
NAME                     REGISTERED   REASON      BUILDSTRATEGYKIND      BUILDSTRATEGYNAME   CREATIONTIME
buildpack-nodejs-build   True         Succeeded   ClusterBuildStrategy   buildpacks-v3       68s
```

Please note we keep the `$` sign in front of the command, so we know what it has been executed, and the outcomes right after. As an additional suggestion, instead of using `bash` as syntax, it should be `text`, as the example above shows.

Reply:
Ah I see now. Making the change 👍🏽

Reply:
Thanks!

Reply:
Via https://github.com/shipwright-io/build/pull/794/commits/48836ab4554933d15c143171090c744482a9cd64
- Put '$' back to command outputs
- Replaced build with *Build* and buildruns with *BuildRuns* since those are names of CRDs.